src/wp-includes/fonts/font-library/class-wp-rest-font-library-controller.php

```diff
@@ -0,0 +1,394 @@
+<?php
```

Comment:
All rest api controllers live in [here](https://github.com/WordPress/wordpress-develop/tree/trunk/src/wp-includes/rest-api/endpoints).

Reply:
:+1: Thanks for flagging this. Moved to the right folder in this commit: https://github.com/WordPress/wordpress-develop/pull/5285/commits/739edeb931c5acf623a05a895eeb6ed0b9147d7c

Reply:
Should the REST API tests also be moved to the `tests/phpunit/tests/rest-api` folder?